addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -122,6 +122,7 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
         // grid padding option so it is not applied on inner rows.
         const $gridPaddingOptions = $html.find('[data-css-property="--grid-item-padding-y"], [data-css-property="--grid-item-padding-x"]');
         $gridPaddingOptions.attr("data-apply-to", ".row.o_grid_mode");
+        $html.find('a[role="tab"], a[role="button"]').addClass('btn');
```

Comment:
You can't do that, especially not in a stable version...
By adding the `btn` class like that, you will change customers designs because suddenly the style of the `.btn` class will be applied.